Japanese/JapaneseAutoUpdate.rc

```diff
@@ -0,0 +1,16 @@
+STRINGTABLE
+LANGUAGE LANG_JAPANESE, SUBLANG_JAPANESE_JP
```

Comment:
It seems like we should use
`LANGUAGE LANG_JAPANESE, SUBLANG_DEFAULT`
instead of SUBLANG_JAPANESE_JP

Could you please update resource files to use the correct header, so I can merge the changes into the correct branch?